.gitignore

```diff
@@ -23,3 +23,5 @@ aula.yaml*
 /static/avatars/*.png
 /hpc
 cookie-jar
+aulaEventLog.json
```

Comment:
that's probably related to something you did to your private aula.yaml?  i would like to keep that out of the repo, but i know i'm not consistent that way.  feel free to ignore me.  (-: